.github/workflows/keepalive.yml

```
@@ -0,0 +1,31 @@
+# 参考: https://qiita.com/kerupani129/items/fda97ed2cf34a156d47d
+name: Prevent scheduled workflow from being automatically disabled
+
+on:
+  # 定期実行
+  schedule:
+    # UTCで実行されているのでJST AM0時過ぎに実行される
+    - cron: "10 15 1 * *"
+
+  # 手動実行
+  workflow_dispatch:
+
+concurrency:
+  group: keepalive
+  cancel-in-progress: false
+
+jobs:
+  # nextjs workflowを実行する
+  keepalive:
+    permissions:
+      actions: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Trigger nextjs workflow
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh workflow run nextjs.yml
```

```suggestion
          gh workflow enable keepalive.yml
          gh workflow enable nextjs.yml
```

`gh workflow run` はActionsを実行するだけで有効化はしないんじゃないですかね 🤔 
参考サイトの通り、`gh workflow enable` する必要があるように思います。
参考: https://cli.github.com/manual/gh_workflow_run

有効化だけではActionsが実行されないなら、一緒にrunも実行する形ですかね 🤔 
（1日動かないだけではどうということは無いと思うので、enableだけで良さそうには思います）

---

おっしゃる通りです。有効化するために定期的に実行させちゃえばいいじゃんって思ってたんですけど、それだと今まで通りで無効化されますね笑